spring-cloud-gcp-docs/src/main/asciidoc/memorystore.adoc

```diff
@@ -0,0 +1,21 @@
+== Cloud Memorystore for Redis
+
+https://cloud.google.com/memorystore/[Cloud Memorystore] for Redis provides a fully managed in-memory data store service
+built on scalable, secure, and highly available infrastructure managed by Google.
+Use Cloud Memorystore to build application caches that provides sub-millisecond data access.
+Cloud Memorystore is compatible with the Redis protocol, allowing easy integration with
+https://spring.io/guides/gs/caching/[Spring Caching] using
+https://spring.io/projects/spring-data-redis[Spring Data Redis].
```

Comment:
I didn't realize that Spring Data Redis was another potential integration.
I would split Caching and Spring Data into two different sections.
Also, I would add a bit more on how to get started. For example, what dependency I need to add to `pom.xml` and what property do I need to set in `application.properties`? Not as comprehensive as the codelab, but the very basics, even if they just show that nothing special needs to be done.